packages/tasks/src/snippets/inputs.ts

```diff
@@ -128,6 +128,7 @@ const modelInputSnippets: {
 	"tabular-classification": inputsTabularPrediction,
 	"text-classification": inputsTextClassification,
 	"text-generation": inputsTextGeneration,
+	"image-text-to-text": inputsTextGeneration,
```

Comment:
this fix was needed, which caused an error on the hub. [slack context](https://huggingface.slack.com/archives/C023JAKTR2P/p1730283278259029?thread_ts=1730282656.555009&cid=C023JAKTR2P).

I've created tests files for snippets that test the snippets for both LLMs & VLMs